database_scripts/tabels/create_address.sql

```diff
@@ -1,13 +1,16 @@
 use customer_db
 
-create table dbo.Address 
+CREATE TABLE [dbo].[Addresses] 
 (
-ID int PRIMARY KEY,
-CustomerID int FOREIGN KEY dbo.Customer,
-Street nvarchar(40),
-City nvarchar(40), PostCodenvarchar(10),
-Country nvarchar(40),
-Level int,
-UpdatedDate datetime,
-CreatedDate datetime
-)
\ No newline at end of file
+    [ID] [int] NOT NULL IDENTITY,
+    [CustomerID] [int],
+    [Street] [nvarchar](max),
+    [City] [nvarchar](max),
+    [Postcode] [nvarchar](max),
+    [Country] [nvarchar](max),
+    [Level] [int],
+    [UpdatedDate] [datetime],
+    [CreatedDate] [datetime],
+    CONSTRAINT [PK_dbo.Addresses] PRIMARY KEY ([ID])
+
+)
```

Comment:
Tu jest ok, tylko musisz dodać NOT NULL przy polach, które są IsRequired() i długości nvarcharów np. `[Street] [nvarchar](100)` oraz zmień `use customer_db` na `use BProject_db`